nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/resources/TestGenerateFakeRecord/nested_no_nullable.avsc

```diff
@@ -0,0 +1,52 @@
+{
+	"type": "record",
```

Comment:
Minor adjustment: recommend reformatting this file using spaces instead of tabs for indentation.